### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gparvin**
effort to stay proactive and ahead of potential CVEs, each squad should
update the libraries and/or packages in their repos for Go and NPM to
the latest versions

Refs:
 - https://github.com/stolostron/backlog/issues/23895

Signed-off-by: Gus Parvin <gparvin@redhat.com>

### CODE REVIEW

**Comment by gparvin** [comment]
/retest
I think this is just a network error

File: go.mod [hunk 1]
```diff
@@ -79,7 +79,6 @@ require (
 )
 
 replace (
-	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220214200702-86341886e292 // CVE-2021-43565
-	k8s.io/client-go => k8s.io/client-go v0.23.3
-	open-cluster-management.io/governance-policy-propagator => github.com/stolostron/governance-policy-propagator v0.0.0-20220427184903-387712d230ee
```
Comment (mprahl) [inline go.mod:84]: You need to keep this replace.

File: go.mod [hunk 2]
```diff
@@ -79,7 +79,6 @@ require (
...
+	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa
```
Comment (mprahl) [inline go.mod:82]: Can you add the CVE comment back so we remember why this is replaced?

Comment (gparvin) [reply go.mod]: Updated

File: go.mod (hunk 1 above)
Comment (gparvin) [reply go.mod:84]: Updated

**Review by mprahl — APPROVED** [review]

**Pull request merged**